src/NServiceBus.AcceptanceTesting/Customization/EndpointConfigurationExtensions.cs

```diff
@@ -4,49 +4,27 @@ namespace NServiceBus.AcceptanceTesting.Customization;
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
-using Features;
 using Hosting.Helpers;
-using Installation;
 using Sagas;
 using Support;
 
 public static class EndpointConfigurationExtensions
 {
     /// <summary>
-    /// Backdoor into the core types to scan. This allows people to test a subset of functionality when running Acceptance tests
-    /// </summary>
-    /// <param name="config">The <see cref="EndpointConfiguration"/> instance to apply the settings to.</param>
-    /// <param name="typesToScan">Override the types to scan.</param>
-    public static void TypesToIncludeInScan(this EndpointConfiguration config, IEnumerable<Type> typesToScan)
```

Comment:
A few downstreams use this still https://github.com/search?q=org%3AParticular%20TypesToIncludeInScan&type=code but they should be changed to `ScanTypesForTest` so that we have a uniform way of doing this.

Should we obsolete with error? (or is it fine to remove since this is not officially supported )